### PR TITLE
allow filtering for errors with sessions

### PR DESCRIPTION
**Pull request by Vadman97**
## Summary

Adds the `HasSession` column to clickhouse error objects for searching for errors with sessions.
Stores the session id on clickhouse error objects in case we have a use for that.

## How did you test this change?

Local deploy
Screenshot 2023-09-26 at 1 37 31 PM

Screenshot 2023-09-26 at 2 03 42 PM

## Are there any deployment considerations?

Will backfill data with https://github.com/highlight/highlight/blob/main/backend/migrations/cmd/backfill-errors-clickhouse/main.go#L1

## Does this work require review from our design team?

Yes @julian-highlight

### CODE REVIEW

File: backend/store/error_groups.go [hunk 1]
```diff
@@ -43,6 +43,11 @@ func (store *Store) ListErrorObjects(errorGroup model.ErrorGroup, params ListErr
 					Where("sessions.email ILIKE ?", "%"+val[0]+"%")
 			}
 		}
+		if val, ok := filters.Attributes["has_session"]; ok {
+			if len(val) > 0 && val[0] == "true" {
+				query.Where("COALESCE(error_objects.session_id) > 0")
```
Comment (mayberryzane) [inline backend/store/error_groups.go:48]: do you want `COALESCE(error_objects.session_id, 0)` here? I think coalesce will still return null if its last argument is null

**Review by mayberryzane — APPROVED** [review]

File: backend/store/error_groups.go (hunk 1 above)
Comment (Vadman97) [reply backend/store/error_groups.go]: thanks! forgot about that

**Comment by julian-highlight** [comment]
Created a quick design here, just to make things look more cohesive: https://www.figma.com/file/rdBPNzn7Klk6RG1LHUCE5B/Screen-Designs?type=design&node-id=12035-22099&mode=design&t=0LzS3Pe1TJMfEp24-11


Low prio to actually implement this from my perspective, as your current solution works as well. This would slightly improve the UX and look of this feature. 
@Vadman97

**Comment by Vadman97** [comment]
thanks @julian-highlight ! noted

**Pull request merged**